statistics/table.go

```diff
@@ -896,3 +896,34 @@ func (coll *HistColl) GetIndexAvgRowSize(ctx sessionctx.Context, cols []*express
 	}
 	return
 }
+
+// CheckAnalyzeVerOnTable checks whether the given version is the one from the tbl.
+// If not, it will return false and set the version to the tbl's.
```

Comment:
Do we assume that this table's all columns and indexes have the same version here?